.github/workflows/smoke.yaml

```
@@ -247,3 +247,55 @@ jobs:
             echo "Docs built incorrectly. Received lines: $(wc -l redoc-static.html) (expected 294 lines in redoc-static.html)."
             exit 1
           fi
+
+  run-smoke-plugins--npm--node-20:
+    needs: prepare-smoke
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/cache@v3
+        with:
+          path: __tests__/smoke-plugins/
+          key: cache-${{ github.run_id }}-${{ github.run_attempt }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+      - run: bash ./__tests__/smoke-plugins/run-smoke.sh
+
+  run-smoke-plugins--npm--node-18:
+    needs: prepare-smoke
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/cache@v3
+        with:
+          path: __tests__/smoke-plugins/
+          key: cache-${{ github.run_id }}-${{ github.run_attempt }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - run: bash ./__tests__/smoke-plugins/run-smoke.sh
+
+  run-smoke-plugins--npm--node-16:
+    needs: prepare-smoke
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/cache@v3
+        with:
+          path: __tests__/smoke-plugins/
+          key: cache-${{ github.run_id }}-${{ github.run_attempt }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - run: bash ./__tests__/smoke-plugins/run-smoke.sh
+
+  run-smoke-plugins--npm--node-14:
+    needs: prepare-smoke
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/cache@v3
+        with:
+          path: __tests__/smoke-plugins/
+          key: cache-${{ github.run_id }}-${{ github.run_attempt }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
```

```suggestion
  run-smoke-plugins--npm--node-22:
    needs: prepare-smoke
    runs-on: ubuntu-latest
    steps:
      - uses: actions/cache@v3
        with:
          path: __tests__/smoke-plugins/
          key: cache-${{ github.run_id }}-${{ github.run_attempt }}
      - uses: actions/setup-node@v3
        with:
          node-version: 22
```

I think it makes more sense to test against the latest Node 😄.

---

Sure. Added jobs for node 22 for all smoke tests